examples/tensorflow-resnet/resources/charts/tensorflow-resnet/README.md

```diff
@@ -0,0 +1,145 @@
+# TensorFlow Serving ResNet
+
+TensorFlow Serving is an open-source software library for serving machine learning models. This chart will specifically serve the ResNet model with already trained data.
+
+## TL;DR;
+
+```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm install my-release bitnami/tensorflow-resnet
+```
+
+## Introduction
+
+This chart bootstraps a TensorFlow Serving ResNet deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment and management of Helm Charts in clusters. This Helm chart has been tested on top of [Bitnami Kubernetes Production Runtime](https://kubeprod.io/) (BKPR). Deploy BKPR to get automated TLS certificates, logging and monitoring for your applications.
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Helm 2.11+ or Helm 3.0-beta3+
```

Comment:
Is there merit in mentioning Helm3 - I don't think we offer helm3 yet so it might be confusing if something breaks b/c it's not backwards (2.x) compatible?